Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM grafana/grafana:7.3.0-ubuntu
+FROM grafana/grafana:7.5.4
 
 USER root
 
-ARG GF_INSTALL_IMAGE_RENDERER_PLUGIN="false"
+ARG GF_INSTALL_IMAGE_RENDERER_PLUGIN="true"
```

Comment:
Eventueel uitleg of link naar docs toevoegen? 

https://grafana.com/docs/grafana/latest/installation/docker/#build-with-grafana-image-renderer-plugin-pre-installed